tests/integration/test_sqlalchemy_integration.py

```diff
@@ -522,3 +522,15 @@ def _num_queries_containing_string(connection, query_string):
     result = connection.execute(statement)
     rows = result.fetchall()
     return len(list(filter(lambda rec: query_string in rec[0], rows)))
+
+
+def test_table_from_other_catalog(trino_connection):
```

Comment:
PTAL at tests results, this test is failing.

Reply:
Thanks for looking.

After poking around, it seems like catalogs aren't currently well supported in the SQLAlchemy dialect and adding support for them will be a much larger change than updating a few methods as I've done here.

Reply:
Hm, maybe it was just the test that I had written incorrectly. Poking around now.